### DISCUSSION TITLE
Does this library support kmp kotlin multiplatform?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [patrykandpatrick/vico#343] [Questions]
See title ^

**Comment by patrickmichalik** [comment]
Hello! At the moment, Vico is Android-only.

**Reply by Gowsky** [reply]
To those who reacted with thumbs down:

1. Vico is free and open-source software.
2. This was a factual answer to a question about the **currently** supported platforms. It was never suggested that Compose Multiplatform support wouldn't be added. In fact, it's planned, as we had already [confirmed](https://github.com/patrykandpatrick/vico/discussions/353).

Instead of responding negatively here for no reason, you can upvote the feature request so that Compose Multiplatform support is prioritized.
